set-web/src/main/java/eu/europeana/set/web/service/impl/BaseUserSetServiceImpl.java

```diff
@@ -518,6 +518,12 @@ private void validateAndSetItems(UserSet storedUserSet, UserSet userSetUpdates)
     if(userSetUpdates.getItems()!=null && userSetUpdates.getItems().size()>0) { 
       storedUserSet.setItems(userSetUpdates.getItems());
     }
+    else {
+      // when we change the type to Gallery, we need to check the items size
```

Comment:
this check must not be in the else, it must be checked every time

Reply:
we need to check again with Hugo if we want to allow the update method to update the items (the code from previous if )